api/envoy/extensions/filters/http/jwt_authn/v3/config.proto

```diff
@@ -277,6 +277,17 @@ message JwtProvider {
   // Enables JWT cache, its size is specified by ``jwt_cache_size``.
   // Only valid JWT tokens are cached.
   JwtCacheConfig jwt_cache_config = 12;
+
+  // Add JWT Claim to HTTP Header
+  // Specify the Claim name you want to copy in which HTTP header. For examples, following config:
```

Comment:
What are the semantics if the header already has a value? append? replace?

Reply:
Replace, if header has any value. No one can set any value to this header, it's just reserved for JWT-Claims

Reply:
The header is sanitized first in the filter.

Reply:
Please use lower case for Claim.  

Please also mention that the header name will be sanitized and replaced.

Reply:
Please explain in the docs that this is replace semantics.